packages/terra-status-view/src/StatusView.jsx

```diff
@@ -252,6 +252,5 @@ class StatusView extends React.Component {
 StatusView.propTypes = propTypes;
 StatusView.contextTypes = contextTypes;
 StatusView.defaultProps = defaultProps;
-StatusView.Opts = {};
```

Comment:
I've been thinking on this and I think it would be preferable to keep this a passive change and add documentation that StatusView.Opts is deprecated and will be removed in a future major version bump. To do that, I think we keep `StatusView.Opts.variants` in addition to exporting `StatusViewVariants`.

e.g.

```jsx
StatusView.Opts = {};
StatusView.Opts.variants = StatusViewVariants;
export default StatusView;
export { StatusViewVariants };
```

Reply:
Updated: [0a2be6a](https://github.com/cerner/terra-core/pull/2264/commits/0a2be6aacc3a2b6c4015aee5729091bb9562a2ab). I also modified the [changelog](https://github.com/cerner/terra-core/pull/2264/commits/47005c670792c89f5279cadb903074eef080d60c) with a note about deprecating the original object; let me know if that could use adjustment.